source/release-notes/4-0-0-rc1.rst

```diff
@@ -0,0 +1,85 @@
+.. index:: Crafter CMS 4.0.0 rc1 Release Notes
+
+---------------------
+Crafter CMS 4.0.0 rc1
+---------------------
+
+Crafter CMS 4.0 rc1 requires Java 11
+
+^^^^^^^^^^^^
+New Features
+^^^^^^^^^^^^
+
+* New Studio UI |br|
+
+  - New React-based UI |br|
```

Comment:
Add sub-bullets:
- Improved Sidebar
- Improved authoring search experience
- Improved user management experience
- Improved code editing
- Improved configuration management
- And more...